civihr_employee_portal/js/ta-documents-app.js

```diff
@@ -124,7 +122,7 @@
            * there must be more that one contacts conidering at aleast a target contact in a document
            */
           $rootScope.$watch('cache.contact', function () {
-            availableContacts = $rootScope.cache.contact.arrSearch.length > 1;
+            availableContacts = $rootScope.cache.contact.arrSearch.length >= 1;
```

Comment:
Apologies, made a mistake in the suggestion. I meant ofc `!$rootScope.cache.contact.arrSearch.length`